SCP Wiki/dist/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+	"author": {
+		"name": "ry",
+		"id": "421371986824921109"
+	},
+	"service": "SCP Wiki",
+	"description": "Operating clandestine and worldwide, the Foundation operates beyond jurisdiction, empowered and entrusted by every major national government with the task of containing anomalous objects, entities, and phenomena.",
+	"url": [
+		"www.scp-wiki.net",
+		"scp-wiki-cn.wikidot.com",
+		"scp-cs.wikidot.com",
+		"fondationscp.wikidot.com",
+		"scp-wiki-de.wikidot.com",
+		"scp-int.wikidot.com",
+		"fondazionescp.wikidot.com",
+		"ja.scp-wiki.net",
+		"ko.scp-wiki.net",
+		"scp-wiki.net.pl",
+		"scp-pt-br.wikidot.com",
+		"scpfoundation.net",
+		"scp-es.com",
+		"scp-th.wikidot.com",
+		"scpclassic.wikidot.com",
+		"scpexplained.wikidot.com",
+		"scp-sandbox-3.wikidot.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://scp-wiki.wdfiles.com/local--files/component%3Atheme/logo.png",
+	"thumbnail": "https://i.imgur.com/el9c68A.png",
+	"color": "#b01",
```

Comment:
Arent color codes 6 long? (hex)

Reply:
They can be both, see https://www.w3.org/TR/CSS21/syndata.html#color-units:

> The format of an RGB value in hexadecimal notation is a '#' immediately followed by either three or six hexadecimal characters. The three-digit RGB notation (#rgb) is converted into six-digit form (#rrggbb) by replicating digits, not by adding zeros. For example, #fb0 expands to #ffbb00. This ensures that white (#ffffff) can be specified with the short notation (#fff) and removes any dependencies on the color depth of the display.

In this case, `#b01` is equivalent to `#bb0011` and is even being used on the SCP Wiki itself:

![image](https://user-images.githubusercontent.com/8181990/68009587-9ce96400-fc82-11e9-85eb-2442c4fd5cdb.png)